entities/backup/descriptor.go

```diff
@@ -103,8 +104,12 @@ func (d *DistributedBackupDescriptor) Include(classes []string) {
 		set[cls] = struct{}{}
 	}
 	pred := func(s string) bool {
-		_, ok := set[s]
-		return ok
+		for cls := range set {
+			if strings.HasPrefix(s, cls) {
+				return true
+			}
+		}
+		return false
```

Comment:
At this point it makes sense to abstract these predicate funcs into their own dedicated named functions

Reply:
Resolved